djangocms_admin_style/static/js/base-admin.js

```diff
@@ -0,0 +1,41 @@
+function toggleClass (element, className) {
+
+    var classString = element.className,
+        nameIndex = classString.indexOf(className);
```

Comment:
`var nameIndex = ..`. We usually start every new variable with a `var`, this convention makes things easier sometimes